src/routes/+server.ts

```
@@ -1,6 +1,14 @@
+import { SETUP_STEPS } from '$lib/common/constants';
+import { getSettings } from '$lib/server/settings/settings';
 import { redirect } from '@sveltejs/kit';
 import type { RequestHandler } from './$types';
 
-export const GET: RequestHandler = ({}) => {
-    return redirect(307, '/dashboard');
+export const GET: RequestHandler = async ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)